### PR TITLE
add more courses vietnamese

### PR DESCRIPTION
**Pull request by tronghieu60s**
## What does this PR do?
Add resources

### Description
Add more resources: React, Vue, Javascript, Android,...

### Why is this valuable (or not)?
I am a Vietnamese student, these are the courses I have learned by myself and I find it easy to understand, I hope to be able to share it with more people.

### How do we know it's really free?
Yes. It's free. All courses are on youtube.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists  in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: courses/free-courses-vi.md [hunk 1]
```diff
@@ -219,6 +239,13 @@
 * [TypeScript căn bản (ES6)](https://tedu.com.vn/khoa-hoc/khoa-hoc-su-dung-typescript-can-ban-9.html) - TEDU
 
 
+### Vue
+
+* [Học Vue js trong một video duy nhất - learn vue js in one video](https://youtu.be/j97QtHf0CHY) - Lập trình viên TV (Bùi Văn Nguyện)
```
Comment (eshellman) [inline courses/free-courses-vi.md:244]: no shortened links like this one

File: courses/free-courses-vi.md [hunk 2]
```diff
@@ -42,14 +44,17 @@
 * [Lập trình Android - Android Widgets - Các điều khiển](https://www.youtube.com/watch?v=2xDWeWllzWg&list=PLv6GftO355At6jjYThbMn-5r164GJ5Vyb) - ZendVN
 * [Lập trình Android - Menu - Context Menu - Dialog](https://www.youtube.com/watch?v=vS_InAnGUFA&list=PLv6GftO355Avjf5iuNbEUsIZbltzDEuIU) - ZendVN
 * [Lập trình Android - Xây dựng bố cục giao diện với Android Layout](https://www.youtube.com/watch?v=oxWDjizEN6g&list=PLv6GftO355AtfPQx7M3dkWgi9KPUB9S0V) - ZendVN
+* [Lập trình Android A-Z](https://www.youtube.com/watch?v=P60kcSaeFmg&list=PL5uqQAwS_KDjAgLGiaCakwJV1f4vRnTLS) - KhoaPham
```
Comment (eshellman) [inline courses/free-courses-vi.md:47]: the corresponding playlist is https://www.youtube.com/playlist?list=PL5uqQAwS_KDjAgLGiaCakwJV1f4vRnTLS These are the kind of youtube links we prefer. I know other links on this page don't follow this rule; they should be changed.

**Review by eshellman — COMMENTED** [review]
Great additions, but we please respect our preferred youtube link style.

File: courses/free-courses-vi.md [hunk 3]
```diff
@@ -39,23 +41,26 @@
...
-* [Lập trình Android cơ bản](https://www.youtube.com/watch?v=rcH4NbngPEk&list=PL33lvabfss1wDeQMvegg_OZQfaXcbqOQh) - Kteam
-* [Lập trình Android với Kotlin](https://www.youtube.com/watch?v=2CKuiSp_uOQ&list=PLzrVYRai0riRFcvx8VYTF7fx4hXbd_nhU) - KhoaPham
+* [Lập trình Android - Android Widgets - Các điều khiển](https://www.youtube.com/playlist?list=PLv6GftO355At6jjYThbMn-5r164GJ5Vyb) - ZendVN
+* [Lập trình Android - Menu - Context Menu - Dialog](https://www.youtube.com/playlist?list=PLv6GftO355Avjf5iuNbEUsIZbltzDEuIU) - ZendVN
+* [Lập trình Android - Xây dựng bố cục giao diện với Android Layout](https://www.youtube.com/playlist?list=PLv6GftO355AtfPQx7M3dkWgi9KPUB9S0V) - ZendVN
+* [Lập trình Android A-Z](https://www.youtube.com/playlist?list=PL5uqQAwS_KDjAgLGiaCakwJV1f4vRnTLS) - KhoaPham
+* [Lập trình Android cơ bản](https://www.youtube.com/playlist?list=PL33lvabfss1wDeQMvegg_OZQfaXcbqOQh) - Kteam
+* [Lập trình Android với Kotlin](https://www.youtube.com/playlist?list=PLzrVYRai0riRFcvx8VYTF7fx4hXbd_nhU) - KhoaPham
+* [Lập trình Android với new Firebase](https://www.youtube.com/playlist?list=PLzrVYRai0riTLPLclyGuByHvZ8_tDZZIr) - KhoaPham
```
Comment (tronghieu60s) [inline courses/free-courses-vi.md:50]: I have changed the appropriate kind of YouTube links that I added, and I also have changed other links.

**Comment by eshellman** [comment]
Outstanding!
Thanks, and welcome to free-programming-books!

**Pull request merged**